collections.go

```diff
@@ -126,6 +127,9 @@ const (
 )
 
 func (cf *CollectionFormat) Ascending() bool {
+	if cf.Reversed {
+		return cf.Format != "novel"
```

Comment:
It would be helpful to have some comments here explaining the logic a bit (or some explanation here on github). I'm not sure why we're checking for the `novel` format here instead of always returning `true`.